Menu/MenuBuilder.php

```diff
@@ -119,7 +119,7 @@ public function createSidebarMenu()
                     }
 
                     $label = $admin->getLabel();
-                    $options = $admin->generateMenuUrl('list');
+                    $options = $admin->generateMenuUrl('list', $item['route_params'], $item['route_absolute']);
```

Comment:
@greg0ire in the current version of the `route_params` was not necessary(it is always created).